tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
```

Comment:
Might want to include `"declaration": true`

Reply:
I will. I removed as you can't use `declaration` with `allowJs` (and I wasn't done migrating everything to TS yet)

Reply:
You can with TypeScript 3.7 😉 https://devblogs.microsoft.com/typescript/announcing-typescript-3-7-beta/

Reply:
One step at a time, Rolee 😅